File: tarea4caso4/Tablero.cpp
#include <stdio.h>
#include <iostream>    
#include <algorithm>   
#include <vector>      

int param_x;
int area;

Tablero(int p_x)
{
    param_x = p_x;
    area = p_x * p_x;
    printf("Se creo tablero")
}

int getX()
{
    return param_x;
}

int getArea()
{
    return area;
}

File: tarea1_analisis_de_algoritmos/Tablero.java
package tarea1_analisis_de_algoritmos;

public class Tablero 
{
    private int param_x;
    private int area;

    Tablero(int p_x)
    {
        param_x = p_x;
        area = p_x * p_x;
    }

    private int getX()
    {
        return param_x;
    }

    private int getArea()
    {
        return area;
    }

}
